src/main/java/com/sonyericsson/jenkins/plugins/bfa/GerritMessageProviderExtension.java

```diff
@@ -98,5 +98,12 @@ private void addFailureCausesFromData(StringBuilder message, FailureCauseDisplay
             .append(displayData.getLinks().getBuildUrl())
             .append(" )");
         }
+
+        if (displayData.getDownstreamFailureCauses().isEmpty() && displayData.getFoundFailureCauses().isEmpty()) {
+            message.append("Unknown (")
```

Comment:
It's too long. UX was pretty bad: small window, a lot of text...

If you want I can create new constant for that or use default message ("No problems were identified. If you know why this problem occurred, please add a suitable Cause for it." aka `DEFAULT_NO_CAUSES_MESSAGE`).